app/core/chat/dst/dst_module.py

```diff
@@ -64,7 +43,7 @@ def remove_special_characters(cls, text: str) -> str:
     def parse_to_dictionary(cls, text: str) -> Dict[str, Any]:
         try:
             data: Dict[str, Any] = json.loads(text)
-            data = {k: data[k] if data[k] != "NA" else None for k in data}
+            data = {k: data[k]for k in data if data[k] != "NA"}
```

Comment:
Good point, forgot to change it after setting `None` as a default value for each slot.